cwd/views/editor.pug

```diff
@@ -0,0 +1,32 @@
+doctype html
+html
+  head
+    title CWD Editor
+    link(href="./public/styleSheet.css" rel="stylesheet" type="text/css")
+
+    body
+      <div id="pass">
+      <p> Password: <input type="password" id="passIn"></input></p>
+      </div>
+
+      <div id="glscButton">
+      <header><b><u>GLSC View Messages</u></b></header>
+      </div>
+
+      BR
+      <div id="lakeButton">
+      <header><b><u>Lake View Messages</u></b></header>
+      </div>
+
+      BR
+      <div id="airButton">
+      <header><b><u>Air View Messages</u></b></header>
+      </div>
+
+      BR
+      <div id="finalButtons">
+      <input id="file-input" type="file" name="name" style="display: none;" />
+      </div>
+
+      script(type="text/javascript" src="http://code.jquery.com/jquery-1.7.1.min.js")
```

Comment:
We should look into packaging jQuery with this application.  I also feel like in an ideal world we wouldn't have to use it (i.e., we are using React).  But I don't want to destroy all the work you have put in.

Reply:
This is now irrelevant because we are moving to React.